pull_request_template.md

```diff
@@ -4,38 +4,39 @@
 - Filling out the template is required. Any pull request that does not include enough information to be efficiently reviewed may be rejected.
 - Please consider which branch this is to be submitted against. This will normally be the development branch, so please ask your reviewer if you think it needs to go somewhere else. 
 -->
+
 ## Type
 
 - [ ] Bug fix
 - [ ] New feature
 - [ ] Improvement to existing feature
 
 ## Related JIRA Issue(s)
-
 <!--_Please provide the URL(s) for any JIRA issues related to this PR._-->
 
-## Description
+## Importance
+<!-- Please explain briefly if this PR specifically needs more importance and also if there is any due date by which it needs to be merged. -->
```

Comment:
I am really skeptical about the wisdom of having this section. For a reviewer to learn that a PR is important, they will first need to read the description of the PR. Which they won't if they are busy with something else. I really think the importance of PRs should be signalled in a different way — either with a label ('urgent') or something, or on a slack channel (we could have a dedicated slack channel for PRs if we don't want to spam the new-website channel with this kind of stuff).

Reply:
Talking to each other (in person on on Slack) before making a PR is important. There's no point in you asking me to review a really important PR by the end of next week if I'm fully committed to something else, or away on holiday. This section of the template would be a prompt for us to consider these things and also a record of what's been decided. An obvious visual indicator of relative importance sounds like a good idea.